example/lib/screens/screens.dart

```diff
@@ -60,6 +74,10 @@ class Example {
     Example(
       title: 'Create token (legacy)',
       builder: (context) => LegacyTokenScreen(),
+    ),
+    Example(
+      title: 'Local paymentMethods',
```

Comment:
Regional Payment Methods might fit better

Reply:
Good one I was thinking for a while how to name it. It is fixed now